packages/environment-check/src/formatter.ts

```diff
@@ -23,3 +23,36 @@ export function countNumberOfServices(catalogResult?: ODataServiceInfo[]): numbe
 export function getServiceCountText(count: number) {
     return count === 1 ? `${count} service` : `${count} services`;
 }
+
+/**
+ * Returns replacer function that can be used with JSON.stringify to detect
+ * and replace circular structures.
+ *
+ * @example JSON.stringify(object, getCircularReplacer());
+ * @returns - replacer that replaces circular structures
+ */
+export function getCircularReplacer(): (key: string, value: any) => any {
```

Comment:
This looks very handy. Do you mind adding a note in the PR description why this is needed?

It's good to have something written down for traceablity

Reply:
Reason is there in the comments already, please refer to: https://github.com/SAP/open-ux-tools/pull/830#discussion_r1068731727

Reply:
It's hard to find comments, especially in resolved conversations. Don't want to nitpick though, resolving